README.md

```diff
@@ -1,137 +1,254 @@
+![](https://img.shields.io/badge/hardware-Nvidia-green.svg?style=popout-square)
 ![](https://img.shields.io/badge/python-3.5%20|%203.6%20|%203.7-blue.svg?style=popout-square)
 ![](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=popout-square)
 
-TensorHive is an open source system for managing and monitoring your computing resources across multiple hosts.
-It solves the most common problems and nightmares about accessing and sharing your AI-focused infrastructure across multiple, often competing users.
+<img src="https://github.com/roscisz/TensorHive/raw/master/images/logo_small.png" height="130" align="left">
+
+TensorHive is an open source system for monitoring and managing computing resources across multiple hosts.
+It solves the most common problems and nightmares about accessing and sharing your AI-oriented infrastructure across multiple, often competing users.
+
+It's designed with __simplicty, flexibility and configuration-friendliness__ in mind.
+
+About project
+----------------------
+Our goal is to provide solutions for painful problems that ML engineers often have to struggle with when working with remote machines in order to run neural network trainings.
+
+#### You should really consider using TensorHive if anything described in profiles below matches you:
+1. You're an **admin**, who is responsible for managing a cluster (or multiple servers) with powerful GPUs installed.
+- :angry: There are more users than resources, so they have to compete for it, but you don't know how to deal with that chaos
+- :ocean: Other popular tools are simply an overkill, have different purpose or require a lot of time to spend on reading documentation, installation and configuration (Graphana, Kubernetes, Slurm)
+- :penguin: People using your infrastructure expect only one interface for all the things related to training models (besides terminal): monitoring, reservation calendar and scheduling distributed jobs 
+- :collision: Can't risk messing up sensitive configuration by installing software on each individual machine, prefering centralized solution which can be managed from one place.
+
+2. You're a **standalone user** who has access to beefy GPUs scatterd across multiple machines.
+- :part_alternation_mark: You want to be able to determine if batch size is too small or if there's a bottleneck when moving data from memory to GPU - charts with metrics such as `gpu_util`, `mem_util`, `mem_used` are great for this purpose
+- :date: Visualizing names of training experiments using calendar helps you track how you're progressing on the project
+- :snake: Launching distributed trainings is essential for you, no matter what the framework is
+- :dizzy_face: Managing a list of training commands for all your distributed training experiments drives you nuts (because you didn't wrap them into bash scripts)
+- :zzz: Remembering to manually launch the training before going sleep is no fun anymore
```

Comment:
```suggestion
- :zzz: Remembering to manually launch the training before going sleep is no fun any more.
```

Reply:
![image](https://user-images.githubusercontent.com/12485656/61938116-9dda6d80-af90-11e9-94e7-6388a7ca0aed.png)

Reply:
OK, thx. I'll leave the dot though :)